### PR TITLE
Edit/customizations variables

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

#554

### Please describe the goal of this pull request and the changes that were made.

- Update Style Customization: add custom classes/ add extend 11ty templates/ update Custom CSS from v0/ add section intro and break into basic, moderate, and advanced
- Update variables.scss guide

### CODE REVIEW

**Review by geealbers — COMMENTED** [review]
I've made some specific comments below. As we spoke about, I didn't get into the weeds on all the details covered on the Variables page, but I think it's fine to publish as is and to post an issue/reminder for us to go back to it another time.

**Comment by Erin-Cecele** [comment]
@geealbers - just pushed the requested edits!

File: content/docs-v1/styles-customization.md [hunk 1]
```diff
@@ -42,19 +42,21 @@ When you start a new Quire project by running the command `quire new`, your proj
 To find the variables, navigate to `content/_assets/styles/variables.scss`. You can also browse a [complete list of these variables](/docs-v1/variables) and the available customization options.
 
 
-The variables are prefixed with a dollar sign and are descriptive of what they control. For instance, `$accent-color` is the background color of the navigation bar at the top of every page. To make it red, you delete `null` and change the value to `red`.
+The variables are prefixed with a dollar sign and are descriptive of what they control. For instance, `$content-background-color` changes the color of the main text area background color. To make it light grey, you delete `null` and change the value to `LightGrey`.
 
 ```css
-$accent-color: red;
+$accent-color: LightGrey;
```
Comment (geealbers) [inline content/docs-v1/styles-customization.md:48]: `$content-background-color: LightGrey`

**Review by geealbers — CHANGES_REQUESTED** [review]
Just one fix

File: content/docs-v1/styles-customization.md (hunk 1 above)
Comment (Erin-Cecele) [reply content/docs-v1/styles-customization.md]: oops! Thank you!

**Comment by Erin-Cecele** [comment]
@geealbers ready for your review again.

**Review by geealbers — APPROVED** [review]

**Pull request merged**